lib/web_ui/lib/src/engine/canvaskit/fonts.dart

```diff
@@ -11,13 +11,32 @@ import '../util.dart';
 import 'canvaskit_api.dart';
 import 'font_fallbacks.dart';
 
-// This URL was found by using the Google Fonts Developer API to find the URL
-// for Roboto. The API warns that this URL is not stable. In order to update
-// this, list out all of the fonts and find the URL for the regular
-// Roboto font. The API reference is here:
-// https://developers.google.com/fonts/docs/developer_api
-const String _robotoUrl =
-    'https://fonts.gstatic.com/s/roboto/v20/KFOmCnqEu92Fr1Me5WZLCzYlKw.ttf';
+// Runtime parameters.
+//
+/// This URL was found by using the Google Fonts Developer API to find the URL
```

Comment:
To use `///` we should have a paragraph before this one that explains what this is (before this paragraph that explains how it was obtained).

Reply:
done.